chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -219,10 +213,16 @@ $(document).ready(function() {
       <td class="cell-meta"><p><a href="${''.join((c['baseUrl'], '?', c['layerBodId'], '=', str(c['featureId']), '&lang=', lang, '&topic=', topic, '&showTooltip=true'))}" target="new">
         ${_('Link to object')}
       </a></p></td>
-  </tr> 
+  </tr>
 </table>
 <br />
 <div>
  <iframe src="${''.join((c['baseUrl'], '/embed.html', '?', c['layerBodId'], '=', str(c['featureId']), '&lang=', lang, '&topic=', topic))}" width='580' height='300' style="width: 100%;" frameborder='0' style='border:0'></iframe>
 </div>
 </%def>
+
+<%def name="extended_resources(c, lang)">
+  <link rel="stylesheet" href="//map.geo.admin.ch/master/2afd257/1610191614/1610191614/style/app.css">
+  <script src="${h.versioned(request.static_url('chsdi:static/js/jquery.min.js'))}"></script>
+  <script src="//map.geo.admin.ch/master/2afd257/1610191614/src/lib/moment-with-customlocales.min.js"></script>
```

Comment:
This is not optimized. The css we use for the stations tooltip are hosted in geoadmin.
Ideally we should migrate them from geoadmin.

Now I wanted to ask you. What is customed in the momentJS lib we have in geoadmin?
If it's nothing we use, we should use a CDN instead.